### PR TITLE
MRG+1: Coreg-GUI: option to skip label

### PR DESCRIPTION
**Pull request by christianbrodbeck**
This adds the option to skip labels and include *.annot files when scaling a brain. Also includes a small layout fix (marked on the screenshot). I wonder whether *.annot files could be linked rather than copied to be more efficient (but there are other files that we just copy during scaling so for now this is more consistent and might be safer)

CC @Eric89GXL 

screen shot 2017-02-20 at 11 25 05 am

### CODE REVIEW

**Comment by Eric89GXL** [comment]
I agree copying is more consistent and safer

**Comment by Eric89GXL** [comment]
While we're cleaning up, can you cherry-pick https://github.com/Eric89GXL/mne-python/commit/7326b34d44b750532a58f01db7ac11f4aa4b11c2?

File: mne/gui/_coreg_gui.py [hunk 1]
```diff
@@ -71,7 +71,8 @@
 
 defaults = DEFAULTS['coreg']
 
-laggy_float_editor = TextEditor(auto_set=False, enter_set=True, evaluate=float)
+laggy_float_editor = TextEditor(auto_set=False, enter_set=True, evaluate=float,
+                                format_func=lambda x: '%0.5f' % x)
```
Comment (Eric89GXL) [inline mne/gui/_coreg_gui.py]: I meant to have this import `_mm_fmt` from `fiducials_gui`, sorry

**Comment by Eric89GXL** [comment]
Set to MRG when you're ready

**Comment by christianbrodbeck** [comment]
Ready if CIs pass

**Comment by Eric89GXL** [comment]
LGTM +1 for merge

Anyone else who wants to feel free to have a look, otherwise I'll merge tomorrow since this one is minor

File: mne/gui/_coreg_gui.py [hunk 2]
```diff
@@ -796,10 +804,21 @@ class CoregPanel(HasPrivateTraits):
...
+                           Item('copy_annot',
+                                label="Copy annotation files",
+                                enabled_when='n_scale_params > 0'),
+                           Item('prepare_bem_model',
+                                label="Run mne_prepare_bem_model",
+                                enabled_when='can_prepare_bem_model'),
+                           show_left=False,
+                           label='Scaling options',
+                           show_border=True,
+                       ),
```
Comment (Eric89GXL) [inline mne/gui/_coreg_gui.py]: Looks like this is not Flake8?

https://travis-ci.org/mne-tools/mne-python/jobs/203536408#L3097

**Comment by Eric89GXL** [comment]
Thanks @christianbrodbeck

**Pull request merged**